packages/grant-explorer/src/features/common/Navbar.tsx

```diff
@@ -13,37 +12,57 @@ export interface NavbarProps {
 export default function Navbar(props: NavbarProps) {
   return (
     <nav className="bg-white">
-        <div className="mx-auto px-4 sm:px-6 lg:px-20">
-            <div className="flex justify-between h-16">
-                <div className="flex">
-                    <Link to="#" className="flex-shrink-0 flex items-center">
-                        <GitcoinLogo className="block h-8 w-auto"/>
-                        <div className="hidden lg:block md:block">
-                            <span className="mx-6 text-grey-400">|</span>
-                            <GrantsExplorerLogo className="lg:inline-block md:inline-block"/>
-                        </div>
-                    </Link>
-                </div>
-                <div className="flex items-center gap-6">
-                    <Search/>
-                    <Shortlist count={0} roundUrlPath={props.roundUrlPath}/>
-                    <ConnectButton/>
-                </div>
-            </div>
+      <div className="mx-auto px-4 sm:px-6 lg:px-20">
+        <div className="flex justify-between h-16">
+          <div className="flex">
+            <Link to="#" className="flex-shrink-0 flex items-center">
+              <GitcoinLogo className="block h-8 w-auto" />
+              <div className="hidden lg:block md:block">
+                <span className="mx-6 text-grey-400">|</span>
+                <GrantsExplorerLogo className="lg:inline-block md:inline-block" />
+              </div>
+            </Link>
+          </div>
+          <div className="flex items-center gap-6">
+            <Search />
+            <Shortlist count={0} roundUrlPath={props.roundUrlPath} />
```

Comment:
This would probably need to be fetched from `loadShortlist(roundId)` and just do a `.length` to it